app/src/components/ImageEditor.tsx

```diff
@@ -30,6 +45,8 @@ function ImageEditor({
   src,
   onImageSave,
   isProfileImage,
+  widthRatio = 50,
+  heightRatio = 50,
 }: imageEditorProps) {
   const handleSave = async (editedImageObject: { imageBase64?: string }) => {
     if (editedImageObject.imageBase64) {
```

Comment:
Why not using react query in the request?

Reply:
This component is a generic tool for image editing. This request converts an image to base64 format. I then use React Query for the requests, such as updating a profile picture or sending stories.